source/index.html.md

```diff
@@ -38,6 +39,22 @@ Pick which SDK you want to use from the tabs on the right (or in the nav on mobi
   Include the videojs-mux plugin after Video.js in the page (or wherever your other Video.js plugins are loaded). If you utilize any ad integrations with Video.js, include the necessary ad integration JavaScript files before the videojs-mux plugin.
 </p>
 
+```objc--shell
+git clone https://github.com/muxinc/stats-sdk-objc.git
+```
+
+```objc--objective_c
+// for iOS
+@import MUXSDKStats;
+
+// for tvOS
+@import MUXSDKStatsTv;
+```
+
+<p class="lang-specific objc">
+  Include the correct Mux Objective-C SDK for your project by cloning our repository and then bringing the right framework into your project. The <code>Frameworks</code> folder contains two folders for iOS and tvOS. Inside these folders, there are 3 additional folders containing different architecture combinations. The <code>fat</code> folder contains a library with all architectures in one. This is library cannot be used when compiling for submission to the App Store as it contains the simulator architectures that are not used by any Apple devices. You can use the framework in the <code>release</code> folder when building a release version of your application, or you can run <a href="https://gist.github.com/brett-stover-hs/b25947a125ff7e38e7ca#file-frameworks_blogpost_removal_script_a-sh">a script to strip unneeded architectures</a>. Finally, don't forget to add the correct import statement for your target platform.
```

Comment:
`The Frameworks folder contains two folders for iOS and tvOS`

to

`The Frameworks folder contains two folders, one for iOS and one for tvOS`